docs/guide/editing_tags.rst

```diff
@@ -108,7 +108,8 @@ and then selecting the appropriate split in the ``Split Tag`` menu.
 There are in general two types of tag splitting possible: splitting on a single
 character and *subtag* splitting (extracting values in enclosures). The
 separating characters for both can be configured in the *Tags* tab in the
-preferences.
+preferences. (Note that the "Split on" values in the configure dialog are space
```

Comment:
Thanks

Can you start this sentence on a newline? Much easier to read that way.

Also that should really be _space-separated_ (with the hyphen)

Reply:
```suggestion
preferences.
(Note that the "Split on" values in the configure dialog are space-separated)
```